content/en/docs/collector/distributions.md

```diff
@@ -22,3 +24,29 @@ functionality like
 [connectors](../building/connector). The tool used to build distributions
 [ocb](../custom-collector) (OpenTelemetry Collector Builder) is available to
 build your own distributions.
+
+## Vendor Distributions
+
+Some organizations provide a Collector distribution with additional capabilities
+or for improved ease of use. What follows is a list of Collector distributions
+maintained by third parties.
+
+{{% alert title="Note" color="warning" %}} OpenTelemetry **does not validate or
+endorse** the third-party distributions listed in the following table. The list
+is provided as a convenience for the community. {{% /alert %}}
+
+{{% ecosystem/distributions-table source="collector-distros" %}}
+
+## Adding your Collector distribution {#how-to-add}
+
+To have your Collector distribution listed, [submit a PR] with an entry added to
+the [distributions list]. The entry should include the following:
+
+- Link to the main page of your distribution
+- Link to the documentation that explains how to use the distribution
+- GitHub handle or email address as a point of contact so that we can reach out
+  in case we have questions
+
+[submit a PR]: /docs/contributing/pull-requests/
+[distributions list]:
```

Comment:
I am _not_ a CI expert, but it seems that maybe the problem is that you're linking to a file that you also created in the same PR. So for the link checker, `collector-distros.yaml` doesn't exist yet and 404s. Maybe remove the link in the docs from this PR and add it in a followup, once this is merged?

Reply:
That worked out!
Thx @tiffany76 🤩